app/src/main/java/org/jellyfin/androidtv/data/service/BlurHashService.kt

```diff
@@ -0,0 +1,35 @@
+package org.jellyfin.androidtv.data.service
+
+import android.content.Context
+import android.graphics.drawable.BitmapDrawable
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import org.jellyfin.androidtv.util.BlurHashDecoder
+
+class BlurHashService(
+	private val context: Context
+) {
+	private val scope = MainScope()
+
+	fun decodeBlurHashDrawable(
```

Comment:
I'd suggest making this function suspended (so no need for the custom `scope`) and pass the viewlifecycle to the `blurHashPlaceholder` function to call the coroutine, this would auto-cancel the loading when the view is destroyed while loading.